### PR TITLE
Fixed grammar mistakes.

### PR DESCRIPTION
**Pull request by MyNameIsTrez**

### CODE REVIEW

File: docs/writing-plugins.md [hunk 1]
```diff
@@ -303,65 +303,65 @@ the event, for example the following could be defined in `info.js`:
...
 
 **broadcastTo**:
-    Target to broadcast this message towards.  Currently only
+    Target to broadcast this message towards.  Currently, only
     `'instance'` is supported and means the event will be broadcast to
     all instances downstream of the target it's sent to, but not back
     from where it came from.  This means that sending an event to a
     slave from an instance will cause it to be broadcast to all
-    instances of that slave except for the instance it came from.
+    instances of that slave, except for the instance it came from.
```
Comment (Hornwitser) [inline docs/writing-plugins.md:337]: The changes in #288 should be added into this PR, otherwise it'll just turn into a merge conflict.  You can add changes to this PR by adding commits to your [patch-2 branch](url=https://github.com/MyNameIsTrez/factorioClusterio/tree/patch-2).

File: docs/writing-plugins.md [hunk 2]
```diff
@@ -303,65 +303,65 @@ the event, for example the following could be defined in `info.js`:
...
-cluster.  For this to work you will need to specify instance-slave,
-slave-master and master-slave, slave-instance as the links.
+cluster.  For this to work, you will need to specify `instance-slave`,
+`slave-master`, `master-slave`, and `slave-instance` as the links.
 
 Keep in mind when forwarding events that if the target an event is being
-forwarded to is not online the event will be dropped.  If you need a
-confirmation that the message was received use a request.
+forwarded to is not online, the event will be dropped.  If you need a
+confirmation that the message was received, use a request.
```
Comment (Hornwitser) [inline docs/writing-plugins.md:355]: I suggest rewording the second sentence to "Use a request if you need a confirmation that the message was received."

File: docs/writing-plugins.md [hunk 3]
```diff
@@ -106,18 +106,18 @@ into plugins is that they must be named the same as the plugin.
 While there is no standard for how to organize a plugin it's recommended
 to put the MasterPlugin class definition into master.js and the
 InstancePlugin class definition into instance.js.  You can put them into
-whatever file you want (even the same one for both)
+whatever file you want (even the same one for both).
```
[the remark this answers is not in the document]
Comment (MyNameIsTrez) [reply docs/writing-plugins.md:109]: I don't mind using your style of writing if it's consistent with the rest of the documents, but I was wondering which was actually 'correct' recently, and people on this website seem to generally agree that a period should go inside the parenthesis and not after:
https://english.stackexchange.com/questions/6632/where-does-the-period-go-when-using-parentheses
Tell me if you still want me to use your writing style.

[the remark this answers is not in the document]
Comment (Hornwitser) [reply docs/writing-plugins.md:109]: I must have mixed up parenthesis rules with quotation rules.  I will change my style to the correct placement (after the parenthesis).
